fix: Add that an incident report requires a previous deployment (#216)
codacy/pulse-user-docs: docs/metrics/accelerate.md

        If you are reporting incidents using [GitHub](../one-click-integrations/github-integration.md) or [Bitbucket](../one-click-integrations/bitbucket-integration.md) integrations, Pulse automatically associates your deployments and incidents with the correct systems.

>     -   Pulse associates an incident to **the last deployment before the incident**, within the same system. If you report an incident and there is no previous deployment for the same system, Pulse doesn't consider it. The same deployment may be associated with multiple incidents.

Reviewer: Tweak to use "with" in both sentences and avoid using "it".

```suggestion
    -   Pulse associates an incident with **the last deployment before the incident**, within the same system. If you report an incident and there is no previous deployment for the same system, Pulse ignores the incident. The same deployment may be associated with multiple incidents.
```

Reviewer 2: The explanation should be the same for 'Change Failure Rate'. Basically what we're saying is that an incident without an associated deployment does not make sense.

Reviewer 2: Sorry I didn't read the 'Change Failure Rate' section which has more specific reasons. The PR is fine as it is.
